packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -691,12 +692,11 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
     required String? flavor,
     required ImpellerStatus impellerStatus,
     required BuildMode buildMode,
+    required String packageConfigPath,
   }) async {
     final AssetBundle assetBundle = AssetBundleFactory.instance.createBundle();
-    // TODO(sigurdm): parametrize packageConfigPath to support testing
-    // workspaces.
     final int build = await assetBundle.build(
-      packageConfigPath: '.dart_tool/package_config.json',
+      packageConfigPath: packageConfigPath,
```

Comment:
I'm sorry - you pointed this mistake out already in the original review.

I think I confused the context somehow, and thought this was a util for the flutter_tools tests, not the test command itself.